Mathlib/Tactic/Positivity/Core.lean

```diff
@@ -242,9 +242,15 @@ def compareHyp (e : Q($α)) (ldecl : LocalDecl) : MetaM (Strictness zα pα e) :
   | ~q(@LE.le _ $_a $lo $hi) =>
     guard <| ← isDefEq e hi
     compareHypLE zα pα lo e (.fvar ldecl.fvarId)
-  | ~q(@LT.lt _ $_a $lo $hi) =>
+  | ~q(@LT.lt.{u} $β $_a $lo $hi) =>
     guard <| ← isDefEq e hi
-    compareHypLT zα pα lo e (.fvar ldecl.fvarId)
+    match ← compareHypLT zα pα lo e (.fvar ldecl.fvarId) with
```

Comment:
I'd like to run the defeq checks first as I think it's probably faster than just running `norm_num`, but it's actually not the easiest as it seems `MetaM` is not a monad that does exceptions. Is there another way to do it easily? Maybe just refactoring this defeq check into some method easily and running `orElse` on it?